src/main/resources/default/templates/biz/cluster/cluster.html.pasta

```diff
@@ -257,7 +257,18 @@
         </t:datacards>
     </i:if>
     <i:if test="locks != null">
-        <t:heading label="Locks"/>
+        <t:heading label="Locks">
+            <i:block name="actions">
+                <i:if test="!locks.getLocks().isEmpty()">
```

Comment:
Maybe extract locks.getLocks() into variable to ensure consistence while rendering